default/stay-calm/tutorial.css

```diff
@@ -0,0 +1,153 @@
+body {
+  background-color: #EAEAEA;
+  color: white;
+  font-family: "Helvetica Neue", Helvetica, sans-serif;
+  padding: 20px;
+}
+
+.tutorial {
+  font-size: 16px;
+  padding: 35px;
+  color: #333;
+  background: #FFEC8D;
+  box-shadow: 0 1px 4px rgba(0,0,0,.2);
+}
+
+h1 {
+    font-size: 2.2em;
+    font-weight: normal;
+    margin: 0 0 15px 0;
+    padding: 0 0 5px 0;
+    border-bottom: solid 2px #C9B45B;
+}
+
+h1,h2,h3,h4,h5,h6 {
+    font-weight: normal;
+    color: rgba(0,0,0,.8);
+}
+
+h2 {
+  font-size: 2em;
+  margin: 35px 0 5px 0;
+}
+
+h3 {
+  font-size: 1.8em;
+  margin: 30px 0 5px 0;
+}
+
+h4 {
+  font-size: 1.5em;
+  margin: 25px 0 5px 0;
+}
+
+h5 {
+  font-size: 1.2em;
+  margin: 20px 0 5px 0;
+  font-weight: bold
+}
+
+h6 {
+  font-size: 1em;
+  font-weight: bold;
+  margin: 20px 0 5px 0;
+}
+
+
+code {
+    background: rgba(0,0,0,.1);
+    font-family: monospace;
+    padding: 1px 3px;
+    margin: 0 1px;
+}
+
+/* Steps list */
+
+ol.steps {
+    list-style-type: none;
+    counter-reset: step-counter;
+    padding-left: 60px;
+    margin: 30px 0 30px 0;
+}
+
+ol.steps > li {
+    position: relative;
+    margin: 0 0 35px 0;
+    min-height: 50px;
+}
+
+ol.steps > li h6 {
+    font-size: 1.5em;
+    font-weight: normal;
+    color: rgba(0,0,0,.6);
+}
+
+ol.steps > li p {
+    line-height: 140%;
+}
+
+ol.steps > li:before {
+    content: counter(step-counter);
+    counter-increment: step-counter;
+    position: absolute;
+    left: -60px;
+    font-size: 16px;
+    border: solid 2px rgba(0,0,0,.4);
+    border-radius: 50%;
+    padding: 10px 15px;
+    opacity: .5;
+    top: 2px;
+}
+
+
+a {
+  color: #357548;
```

Comment:
Can you normalize the indents in this file?  In this section you go back to 2 spaces.